examples/reference_environments/postgres/docker-compose.yaml

```diff
@@ -0,0 +1,41 @@
+services:
```

Comment:
Are the file suffixes supposed to be `.yaml` or `.yml`?  In our existing `assets/docker` they are the latter.  Thanks.

Reply:
Good question! `.yaml` is preferred for `docker-compose.yaml` but either is OK: https://docs.docker.com/compose/compose-file/03-compose-file/. But it looks like `compose.yaml` is actually the preferred, so I may rename to that.

Reply:
Renamed to the new preferred filename `compose.yaml` 🙇